packages/node_modules/@webex/internal-plugin-device/src/device.js

```diff
@@ -651,9 +654,25 @@ const Device = WebexPlugin.extend({
     delete body.services;
     delete body.serviceHostMap;
 
+    const {etag} = response.headers;
+
+    if (this.etag && this.etag === etag) {
+      // If current etag matches the previous one and we have sent
+      // If-None-Match header the developer and entitlement feature
+      // toggles will not be returned
+      const {features} = body;
+
+      delete body.features;
+      // When using the etag feature cache, user features are still returned
+      this.features.user.reset(features.user);
+    }
+
     // Assign the recieved DTO from **WDM** to this device.
     this.set(body);
 
+    // Assign the new etag to this device.
+    this.set({etag});
```

Comment:
suggestion: Do these together (reduces the number of change events potentially):

```js
this.set({...body, etag: response.headers.etag});
```

I removed the destructuring because it's only used here, but obviously it can stay if wanted.